documentation/assemblies/metrics/assembly-metrics-config-files.adoc

```diff
@@ -39,12 +39,17 @@ metrics
 │    ├── prometheus-rules.yaml <7>
 │    ├── prometheus.yaml <8>
 │    └── strimzi-pod-monitor.yaml <9>
-├── kafka-bridge-metrics.yaml <10>
-├── kafka-connect-metrics.yaml <11>
-├── kafka-cruise-control-metrics.yaml <12>
-├── kafka-metrics.yaml <13>
-├── kafka-mirror-maker-2-metrics.yaml <14>
-└── oauth-metrics.yaml <15>
+├── strimzi-metrics-reporter
+|    ├── grafana-dashboards <10>
+|    |  ├── strimzi-kafka.json
+|    |  ├── strimzi-kraft.json
+|    └── kafka-metrics.yaml <11>
+├── kafka-bridge-metrics.yaml <12>
+├── kafka-connect-metrics.yaml <13>
+├── kafka-cruise-control-metrics.yaml <14>
+├── kafka-metrics.yaml <15>
+├── kafka-mirror-maker-2-metrics.yaml <16>
+└── oauth-metrics.yaml <17>
```

Comment:
a couple of questions here:

* aren't we getting examples for all the other components (apart Kafka) for using the Strimzi Metrics reporter?
* wouldn't be better to put the Prometheus JMX exporter related example in a dedicated folder now that we have the `strimzi-metrics-reporter` one?

Reply:
Yeah that was the idea thanks @OwenCorrigan76 
@fvaleri @PaulRMellor wdyt?

Reply:
LGTM, but the folder name should be `prometheus-jmx-exporter` and we need a similar README.md for consistency.

Reply:
Yes. I'll update this docs PR and then update the main integration PR. Thanks